README.md

```diff
@@ -15,41 +15,61 @@ Files for running SciCat with docker-compose.
    ```
 3. SciCat will now be available on http://localhost. The Loopback API explorer of the backend is available at http://localhost/explorer/, the one for the search-api at http://localhost/panosc-explorer/.
 
-## Add Your Local Configuration
+## Default setup
 
-1. Add your local configuration to [config.local.js](./config/backend/config.local.js)
-2. Uncomment the `volumes:` line and the line containing `config.local.js` in the backend service section in [docker-compose.yaml](./docker-compose.yaml) (if commented)
-3. Restart the docker containers
+By running `docker-compose up -d` these steps take place: 
+1. a [mongodb](./service/mongodb/) container is created.
+2. some initial data is [imported](./service/mongodbseed/) in (1).
+3. the SciCat [backend v3](./service/backend/) container is created and connected to (1).
+4. the SciCat [frontend](./service/frontend/) container is created and connected to (3).
+5. the SciCat [PaN searchapi](./service/searchapi/) container is created and connected to (3).
 
 
-## Add LDAP Authentication
+```mermaid
+graph TD;
+   mongodb --- mongodbseed;
+   mongodb --- backend;
+   backend --- frontend;
+   backend --- searchapi;
+```
 
-1. Add your LDAP configuration to [providers.json](./config/backend/providers.json)
-2. Uncomment the `volumes:` line and the line containing `providers.json` in the backend service section in [docker-compose.yaml](./docker-compose.yaml)
-3. Restart the docker containers 
+## Select the services
 
+The user can selectively decide the containers to spin up and the dependencies will be resolved accordingly. The available services are in the [service](./service/) folder and called consistently. 
 
-## Functional Accounts
 
-There are a few functional accounts available for handling data:
+For example, one could decide to only run the `backend` by running:
 
-| Username         | Password    | Usage                        |
-| ---------------- | ----------- | ---------------------------- |
-| admin            | 2jf70TPNZsS | Admin                        |
-| ingestor         | aman        | Ingest datasets              |
-| archiveManager   | aman        | Manage archiving of datasets |
-| proposalIngestor | aman        | Ingest proposals             |
+```sh
+docker-compose up -d backend
```

Comment:
do the traefik labels have an affect here? Also the dependency on mongo ? I can't see these being used in the compose for the backend.